backend/docker-compose.yaml

```
@@ -1,10 +1,28 @@
 version: "3.7"
+
+volumes:
+  postgres_data:
+    driver: local
+
 services:
 
+  postgres:
+    image: postgres:11.7
+    volumes: 
+      - postgres_data:/var/lib/postgresql/data
+    environment: 
+      POSTGRES_DB: housinghub
+      POSTGRES_USER: app
+      POSTGRES_PASSWORD: apppassword
+    ports:
+      - 5432:5432
+
   graph:
```

```suggestion
  housinghub-backend
```
This should be changed.. my bad, ported this from another project